app/utils/bundle-importer.js

```diff
@@ -489,6 +493,21 @@ YUI.add('bundle-importer', function(Y) {
           }.bind(this),
           {modelId: relation.get('id')});
       next();
+    },
+
+    /**
+      Executes the setAnnotations method call
+
+      @method _execute_setAnnotations
+      @param {Object} record The setAnnotations record.
+      @param {Function} next The method to trigger the executor to move
+        on to the next record.
+    */
+    _execute_setAnnotations: function(record, next) {
```

Comment:
None of these executors have any tests. They were in too much flux and buggy to write tests on their implementations. I have one more bug to fix then there will be a tests branch.